### ISSUE TITLE
 As a website owner, I can set small, medium, and large image sizes within a card and have them use corresponding Drupal image styles

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#2475]
# Story

As a website owner I want to be able to set small, medium, and large image sizes within the card block and have them use corresponding image styles.

# Proposed solution

Update the card to use small, medium, and large image styles based on the image size selection.

**Comment by joewhitsitt** [comment]
If the cache issue is not a long term solution, I think something like this might have to be done: https://www.flocondetoile.fr/blog/generate-programmatically-image-styles-drupal-8

**Comment by bspeare** [comment]
Current image styles are being set at: https://github.com/uiowa/uiowa/blob/main/docroot/modules/custom/layout_builder_custom/src/EventSubscriber/SectionComponentSubscriber.php

**Comment by joewhitsitt** [comment]
I'd like to see this functionality on image gallery in the future

**Comment by pyrello** [comment]
@bspeare Is this completed?

**Comment by bspeare** [comment]
No, we are sizing the images with CSS and the same image style (`/admin/config/media/image-styles`) loads for all sizes.

**Comment by pyrello** [comment]
Ah. I was misreading this issue. Thanks!

**Comment by bspeare** [comment]
I added "Drupal" to the issue title to help clarify that :)

**Comment by bspeare** [comment]
I think I'd rather prioritize https://github.com/uiowa/uiowa/issues/5432#issuecomment-1219591778 over this issue if we had to choose an issue to improve our responsive image experience.
